.gitignore

```diff
@@ -0,0 +1,32 @@
+# Helm-specific ignores
+**/charts/*.tgz
+
+# Ignore Helm dependency directories
+**/charts/*.lock
+
+# Ignore saved Helm values files
+**/helm/values-*.yaml
```

Comment:
That's a strange ignore, I'lll look through the rest but why are you ignoring values?

Reply:
copy/paste or Copilot going rouge probably